sycl/unittests/queue/ShortcutFunctions.cpp

```diff
@@ -0,0 +1,238 @@
+//==-------------- ShortcutFunctions.cpp --- queue unit tests --------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#include <detail/context_impl.hpp>
+#include <gtest/gtest.h>
+#include <helpers/PiMock.hpp>
+#include <sycl/ext/oneapi/accessor_property_list.hpp>
+#include <sycl/handler.hpp>
+#include <sycl/queue.hpp>
+#include <sycl/sycl.hpp>
+
+#include <memory>
+
+using namespace sycl;
+
+namespace {
+struct TestCtx {
+  TestCtx(context &Ctx) : Ctx{Ctx} {};
+
+  context &Ctx;
```

Comment:
Could you please clarify why the context is needed here?

Reply:
Oh, I'd popped a stashed piece of work that had this change in addition to the context object. This isn't required, so I'll back that change out. Apologies.